untitled/main/com/aca/homework/week11/repository/invoice/Invoice.java

```diff
@@ -0,0 +1,33 @@
+package com.aca.homework.week11.repository.invoice;
+
+public class Invoice {
```

Comment:
the equals and hashcode are not implemented. because of that the hashSet will not work for Invoice